lib/static/styles.css

```diff
@@ -979,6 +979,6 @@ a:active {
     color: var(--g-color-private-cool-grey-600-solid);
 }
 
-.icon-retry {
+.icon-retry, .icon-updated {
```

Comment:
why are these colors the same?

Reply:
Now it's pink:

<img width="273" alt="Screenshot 2024-09-17 at 19 00 14" src="https://github.com/user-attachments/assets/1a35044b-534a-44e7-8989-c3476376f407">